components/Main/Exercises/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,123 @@
+import styled from "styled-components";
+
+export default function SearchBar({ onChange }) {
+  function handleSearchClick() {
+    if (!document.getElementById("search-bar").focus()) {
+      document.getElementById("search-bar").blur();
```

Comment:
🤔

Reply:
Changed the function. hope its more comprehensive now